src/components/Skeletons/DataOverview.vue

```diff
@@ -100,22 +100,22 @@
               <span>
                 {{ row.totalUpdates }}
               </span>
-              <span
-                v-if="row.totalRejectedUpdates"
-              >
-                <KPop
-                  placement="top"
-                  trigger="hover"
-                >
-                  <KIcon
-                    class="kong-icon--centered"
-                    color="var(--yellow-300)"
-                    icon="warning"
-                    style="margin:auto"
-                  />
-                  <div slot="content"><b>{{ row.totalRejectedUpdates }} rejection(s) </b>. Some configuration changes were rejected by Envoy. The most common reason is that the configuration generated by the control plane may be invalid. Check {{ $productName }} DP sidecar logs for more details.</div>
-                </KPop>
-              </span>
+<!--              <span-->
```

Comment:
Can we maybe put here a link to the issue on github, to avoid having commented code without any explanation?